content/changelog-stable/index.html.haml

```diff
@@ -11,22 +11,31 @@ actions:
 ---
 
 .ratings
-  - # source: https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/lts.yml
-  - site.changelogs[:lts].reverse_each do | release |
+  - # Load the most recent 25 changelogs (latest first)
+  - recent_changelogs = site.changelogs[:lts].reverse.take(27)
```

Comment:
Same as above: Is there a reason why the `27` is hard-coded?

Reply:
I agree with @krisstern that 25 is more than enough in regards to what should be shown. The oldest Jenkins LTS version that is typically encouraged/supported would be included in this without having too many additional versions displayed.